shared-data/deck/schemas/2.json

```
@@ -112,7 +112,7 @@
         "model": {
           "description": "Model of the robot (currently only OT-2 Standard is supported)",
```

Looks like this description needs to change too.
```suggestion
          "description": "Model of the robot",
```